datastore/db/management/commands/create_data_package.py

```diff
@@ -86,3 +112,86 @@ def flatten_grant(in_grant):
             data_all_fp.write(json.dumps(data_all, indent=options["indent"]))
 
         spinner.stop()
+
+
+CREATE_RELATED_ORGIDS = """
+DROP TABLE IF EXISTS related_orgids;
+
+CREATE TABLE related_orgids AS (    
```

Comment:
Would it be worth making this a "CREATE TEMP" so that we're not making public a table which the persistence shouldn't be relied upon?